packages/vite/src/node/server/transformRequest.ts

```diff
@@ -240,9 +240,11 @@ async function loadAndTransform(
 
         code = code.replace(convertSourceMap.mapFileCommentRegex, blankReplacer)
       } catch (e) {
-        logger.warn(`Failed to load source map for ${url}.`, {
-          timestamp: true,
-        })
+        if (server.config.build.sourcemap) {
```

Comment:
I think `config.build.sourcemap` only applies during build. Is there actually a way to disable sourcemaps during dev?

Reply:
I don't think dev has an option, it's always implied to be enabled. I'm not really sure if we need an option, or use this build option though. It seems like a helpful warning, but if not you could use customLogger to filter it out.

Reply:
Yeah. But I still think it might be better to maybe have an dev option to disable the warning. CustomLogger would do for Vite users, but it could be hard for the meta framework to do so.

Either way let's fix the upstream issue first: https://github.com/inikulin/parse5/issues/831